<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaspForm</class>
 <widget class="QWidget" name="RaspForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1153</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Дата</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Начало</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Окончание</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QDateEdit" name="dateEdit"/>
         </item>
         <item row="1" column="1">
          <widget class="QTimeEdit" name="timeEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QTimeEdit" name="timeEdit_2"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Распоряжение выдал</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_3"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <column>
          <property name="text">
           <string>Обозначение</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Тип</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Вид работ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Помещение</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_2">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <column>
          <property name="text">
           <string>Состав бригады</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_3">
           <property name="text">
            <string>&lt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="toolButton_4">
           <property name="text">
            <string>&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="splitterButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>&gt;
&gt;
&gt;
&gt;
&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Блок</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="unitBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Месяц</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="monthBox">
           <item>
            <property name="text">
             <string>Весь год</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Январь</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Февраль</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Март</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Апрель</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Май</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Июнь</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Июль</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Август</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Сентябрь</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Октябрь</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Ноябрь</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Декабрь</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Статус</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="NewCheckBox">
            <property name="text">
             <string>Новые</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="InworkCheckBox">
            <property name="text">
             <string>В работе</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="CompletedCheckBox">
            <property name="text">
             <string>Завершённые</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="workTable">
       <column>
        <property name="text">
         <string>Обозначение</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Тип</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Вид работ</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Помещение</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Ок</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
